ensembl/htdocs/info/genome/compara/homology_method.html

```diff
@@ -132,12 +132,19 @@ <h2 id="treebest">Tree building</h2>
 support. Branch lengths are estimated for the final consensus tree
 based on the DNA alignment, using phyml with the HKY model.</p>
 
+
+<h2 id="breedstrees">Gene tree building in Murinae and Pig breeds</h2>
+
+<p>A modified approach is used to generate most protein trees for strain and breed gene-tree datasets  <a href="#fn7">[7]</a><a href="#fn8">[8]</a>. Under this approach, a phylogeny is inferred on a cDNA alignment from which problematic columns have been removed using Noisy  <a href="#fn9">[9]</a>(version 1.5.12). Gene trees are inferred using RAxML  <a href="#fn10">[10]</a> (version 8.2.12) or ExaML  <a href="#fn11">[11]</a> (version 3.0.22) from a RAxML parsimony starting tree, all under a GTR+Γ model. If gene-tree inference requires excessive runtime, we fall back to FastTree  <a href="#fn12">[12]</a> (version 2.1.8) with the GTR model.</p>
```

Comment:
This suggestion is smaller than it looks, and is just to add a couple of spaces.
```suggestion
<p>A modified approach is used to generate most protein trees for strain and breed gene-tree datasets  <a href="#fn7">[7]</a> <a href="#fn8">[8]</a>. Under this approach, a phylogeny is inferred on a cDNA alignment from which problematic columns have been removed using Noisy  <a href="#fn9">[9]</a> (version 1.5.12). Gene trees are inferred using RAxML  <a href="#fn10">[10]</a> (version 8.2.12) or ExaML  <a href="#fn11">[11]</a> (version 3.0.22) from a RAxML parsimony starting tree, all under a GTR+Γ model. If gene-tree inference requires excessive runtime, we fall back to FastTree  <a href="#fn12">[12]</a> (version 2.1.8) with the GTR model.</p>
```